### PR TITLE
New Serverless Pattern (CDK - Python) - Apigw - lambda - bedrock

### PR DESCRIPTION
**Pull request by gmansilla**
Pattern to create an endpoint that receives a prompt for text generation on Amazon Bedrock.

https://github.com/aws-samples/serverless-patterns/issues/1699

### CODE REVIEW

File: apigw-lambda-bedrock-cdk-python/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+1. Change directory to the pattern directory:
+    ```
+    cd apigw-lambda-bedrock-cdk-python
+    ```
+1. From the command line, use AWS CDK to deploy the AWS resources.
+    ```
+    cdk deploy
```
Comment (mavi888) [inline apigw-lambda-bedrock-cdk-python/README.md:30]: Error when following the instructions:
```
Traceback (most recent call last):
  File "/apigw-lambda-bedrock-cdk-python/app.py", line 4, in <module>
    import aws_cdk as cdk
ModuleNotFoundError: No module named 'aws_cdk'
```

File: apigw-lambda-bedrock-cdk-python/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+CDK will create an Api Gateway, along with a resource and a POST method. There's a AWS Lambda function that will be taking the prompt and invoking an Amazon Bedrock model (anthropic.claude-v2). If you wish to try other models, make sure to modify the policy attached to the Lambda function and invoke the right model.
+
+## Testing
+
+After deployment, take note of the API Gateway URL (Check the Outputs section) and make a POST request to the *text_gen* endpoint with a desired prompt. For example:
+
+    ```
+    curl -X POST \
+    https://ygah153dth.execute-api.us-east-1.amazonaws.com/prod/text_gen \
```
Comment (mavi888) [inline apigw-lambda-bedrock-cdk-python/README.md:43]: is this a real api gateway url? has it been deleted already? please put a place holder like xxx.execute-api....

File: apigw-lambda-bedrock-cdk-python/README.md (hunk 1 above)
Comment (gmansilla) [reply apigw-lambda-bedrock-cdk-python/README.md:75]: updated

File: apigw-lambda-bedrock-cdk-python/README.md (hunk 2 above)
Comment (gmansilla) [reply apigw-lambda-bedrock-cdk-python/README.md]: Updated

**Pull request merged**